Deploy

balmas opened the pull request:
this change deploys all branches to 

https://alpheios-misc-dev.s3.us-east-2.amazonaws.com/alignment-editor/public/index.html

We can further restrict it by branch as needed but at this stage maybe all branches is okay?

It currently deploys the entire directory because the public dir references files in dist. It would probably be better to have a script which produces a public dir with the dist directory below it and to deploy just that, because there isn't any need to deploy all the source and node_modules.  That would be a good change to make in an upcoming PR

irina060981 commented on src/vue/align-editor/align-editor-single-block.vue:
```diff
@@ -7,7 +7,7 @@
         <token
           v-if ="token.word"
-          :text-type = "textType" :text-word = "token"
+          :text-type = "textType" :text-word = "token" :key = "token.idWord"
```
it is a tiny fix for @kirlats comment 
https://github.com/alpheios-project/alignment-editor-new/pull/10#discussion_r468583873

irina060981 commented on .github/workflows/workflow.yml:
```diff
@@ -0,0 +1,29 @@
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'us-east-2'
+        SOURCE_DIR: '.'
+        DEST_DIR: 'alignment-editor/public'
```
changed to public dir

irina060981 commented:
@balmas, I have updated build with placing dist inside public folder.
And changed workflow.yml to upload only public

irina060981 commented on .github/workflows/workflow.yml:
```diff
@@ -0,0 +1,29 @@
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'us-east-2'
+        SOURCE_DIR: 'public'
```
changed to public dir

balmas commented:
> @balmas, I have updated build with placing dist inside public folder.
> And changed workflow.yml to upload only public
> 
> Bridget, could you check? I am not sure - if I do everything right.

Yep, looks good. You can see that it deployed at https://alpheios-misc-dev.s3.us-east-2.amazonaws.com/alignment-editor/index.html

(You can see the results of GitHub Actions on the Actions tab)

irina060981 commented:
great :) than I believe I could merge it our joint work :)
